### DISCUSSION TITLE
V4 Support

### DISCUSSION THREAD

**Discussion by jhagan-91** [tighten/novapackages#213] [General]
How often will it check my package to check and display it supports v4

**Comment by marcusmoore** [comment]
Hi @itsmejoshua,

The app checks if `laravel/nova` with `4` in the major version is included in the `require` section of the package's composer file to determine Nova 4 support.

So NovaPackages.com would show your package having Nova 4 support if you were to tag a new release that explicitly required `laravel/nova` and waited about an hour for the app to pick it up.

Let me know if I can answer any other questions 😄

**Comment by jhagan-91** [comment]
Sure I added to GitHub yesterday the require and tagged a new release for today so it would update at the package site just not seeing it come through 

https://github.com/itsmejoshua/novaspatiepermissions is the package in question please kindly let me know if I am missing anything.

Appreciate your help.

**Comment by marcusmoore** [comment]
I missed a step in the process: we also have to wait for Packagist to pick up the change 😅 

Looks like they're still showing `v1.0.5` as the latest. I'm not sure how often they update their index but I would be surprised if it hasn't updated by this time tomorrow.

![CleanShot 2022-05-16 at 21 11 00](https://user-images.githubusercontent.com/1141514/168727399-17f1fb36-4397-42c7-a51c-6488bd606f2b.png)

https://packagist.org/packages/itsmejoshua/novaspatiepermissions

**Reply by jhagan-91** [reply]
> I missed a step in the process: we also have to wait for Packagist to pick up the change 😅
> 
> Looks like they're still showing `v1.0.5` as the latest. I'm not sure how often they update their index but I would be surprised if it hasn't updated by this time tomorrow.
> 
> https://packagist.org/packages/itsmejoshua/novaspatiepermissions

Makes sense to me now thanks for clarifying that.

**Reply by marcusmoore** [reply]
![CleanShot 2022-05-18 at 17 08 01](https://user-images.githubusercontent.com/1141514/169175069-73294f39-d84e-4ce9-9201-2def564f6356.png)

😄
